buildspec.json

```diff
@@ -49,7 +49,7 @@
         }
     },
     "name": "obs-ndi",
-    "version": "4.14.0",
+    "version": "4.13.1",
```

Comment:
Change this to 4.14.1, or preferably just leave alone.
This is on the develop branch and I will increment the version number to 4.15.0 anyway when actually released to master.